src/components/FilterHeader/FilterHeader.css

```diff
@@ -19,3 +21,17 @@
   margin-right: 20px;
   padding: 0 20px 0 0;
 }
+
+input {
```

Comment:
> A [CSS Module](https://github.com/css-modules/css-modules) is a CSS file in which all class names and animation names are scoped locally by default. All URLs (url(...)) and @imports are in module request format (./xxx and ../xxx means relative, xxx and xxx/yyy means in modules folder, i. e. in node_modules).

Be careful with styling all inputs in a CSS module. I haven't checked, but I think this will either:

1. not work
2. style all inputs globally with this style

Use a class, and then pass this guy to the `className` property.